src/validation-errors/__tests__/main.test.js

```diff
@@ -1,4 +1,4 @@
-import Ajv from 'ajv';
+import Ajv from 'ajv/dist/2020';
```

Comment:
`unevaluatedProperties` is new to the 2020 draft so for this new test to work I needed to change this import.